src/sparseml/exporters/transforms/utils/helpers.py

```diff
@@ -161,3 +161,115 @@ def assert_node_type(node: NodeProto, op: Union[List[str], Set[str], str]) -> bo
         return node.op_type == op
     else:
         return node.op_type in op
+
+
+NodeOrInit = Union[NodeProto, TensorProto]
```

Comment:
maybe a short comment here? it is not common knowledge that `TensorProto` in the context of exporters will usually describe an initializer